src/Dapr.Actors/Runtime/ActorStateManager.cs

```diff
@@ -21,7 +21,7 @@ internal sealed class ActorStateManager : IActorStateManager
         internal ActorStateManager(Actor actor)
         {
             this.actor = actor;
-            this.actorType = actor.GetType().Name;
```

Comment:
I understand its not part of this PR, but could you please rename actorType member variable to actorTypeName also with this PR as it seems more logical.

Reply:
Updated.